lib/rule-interface/converter.rb

```diff
@@ -2,18 +2,22 @@ module RuleInterface
   module Converter
     extend self
 
-    def hash_to_drool(data_hash:, namespace:, package:)
+    def hash_to_drool(data_hash:, namespace:, package:, session:)
       payload = {}
-      payload[:lookup] = 'session'
+      payload[:lookup] = session
       payload[:commands] = []
 
+      Thread.current[:id] = 1
```

Comment:
Move `:id` to a constant and rename the value to `:rule_interface_out_identifier_id`